.github/workflows/ios-demos.yml

```diff
@@ -0,0 +1,100 @@
+name: iOS Demos
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ master ]
+    paths:
+      - 'demo/ios/**'
+      - '.github/workflows/ios-demos.yml'
+  pull_request:
+    branches: [ master, 'v[0-9]+.[0-9]+' ]
+    paths:
+      - 'demo/ios/**'
+      - '.github/workflows/ios-demos.yml'
+
+jobs:
+  build-foreground-app:
+    runs-on: macos-latest
+    defaults:
+      run:
+        working-directory: demo/ios/ForegroundApp
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - name: Set up Node.js LTS
+        uses: actions/setup-node@v3
+        with:
+          node-version: lts/*
+
+      - name: Install Cocoapods
+        run: gem install cocoapods
+
+      - name: Install AppCenter CLI
+        run: npm install -g appcenter-cli
+
+      - name: Make build dir
+        run: mkdir ddp
+
+      - name: Run Cocoapods
+        run: pod install
+
+      - name: Build English
+        run: xcrun xcodebuild build
+          -configuration Debug
+          -workspace PicovoiceForegroundAppDemo.xcworkspace
+          -sdk iphoneos
+          -scheme enDemo
+          -derivedDataPath ddp
+          CODE_SIGNING_ALLOWED=NO
+
+      - name: Build Other Language
+        run: xcrun xcodebuild build
+          -configuration Debug
+          -workspace PicovoiceForegroundAppDemo.xcworkspace
+          -sdk iphoneos
+          -scheme deDemo
+          -derivedDataPath ddp
+          CODE_SIGNING_ALLOWED=NO
+
+  build-background-service:
+    runs-on: macos-latest
+    defaults:
+      run:
+        working-directory: demo/ios/BackgroundService
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - name: Set up Node.js LTS
+        uses: actions/setup-node@v3
+        with:
+          node-version: lts/*
+
+      - name: Install Cocoapods
+        run: gem install cocoapods
+
+      - name: Install AppCenter CLI
+        run: npm install -g appcenter-cli
+
+      - name: Make build dir
+        run: mkdir ddp
+
+      - name: Run Cocoapods
+        run: pod install
+
+      # - name: Build
+      #   run: xcrun xcodebuild build
+      #     -configuration Debug
+      #     -workspace PicovoiceBackgroundServiceDemo.xcworkspace
+      #     -sdk iphoneos
+      #     -scheme PicovoiceBackgroundServiceDemo
+      #     -derivedDataPath ddp
+      #     CODE_SIGNING_ALLOWED=NO
```

Comment:
I have a task to audit this demo later, seems like it is broken right now. I will make a different PR for this part as it is a different task.